apiserver/src/main.rs

```diff
@@ -1,3 +1,33 @@
-fn main() {
-    println!("Hello, world!");
+use apiserver::api::{self, APIServerSettings};
+use apiserver::error::{self, Result};
+use models::node::K8SBottlerocketNodeClient;
+
+use snafu::ResultExt;
+
+use std::fs;
+
+#[actix_web::main]
+async fn main() {
+    match run_server().await {
+        Err(error) => {
+            fs::write("/dev/termination-log", format!("{}", error))
+                .expect("Could not write k8s termination log.");
+        }
+        Ok(()) => {}
+    }
+}
+
+async fn run_server() -> Result<()> {
+    env_logger::init();
+
+    let k8s_client = kube::client::Client::try_default()
+        .await
+        .context(error::ClientCreate)?;
+
+    let settings = APIServerSettings {
+        node_client: K8SBottlerocketNodeClient::new(k8s_client),
+        server_port: 8080,
```

Comment:
Maybe use an env variable or mount configuration file to fetch the port in which the server has to run?

Reply:
In this case, this is the port used inside the server container itself, which needs to be used in the kubernetes deployment files as well. It should probably be a constant so that it's easy to cross reference through `brupop's` repo, but I don't think there's a use-case which would require changing it: Folks can use port mapping at the cluster level to decide which port is actually used for communication.